src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -41,7 +41,7 @@ bool ClusterTaskManager::SchedulePendingTasks() {
       continue;
     } else {
       if (node_id_string == self_node_id_.Binary()) {
-        did_schedule = did_schedule || WaitForTaskArgsRequests(work);
+        did_schedule = WaitForTaskArgsRequests(work) || did_schedule;
```

Comment:
Is the ordering of these statements important (why did you reverse them)? If it is, I'd leave a comment as that could be a gotcha for someone in the future

Reply:
If it is important, you might consider calling `WaitForTaskArgsRequests` on a separate line instead of using the bool evaluation order for clarity

Reply:
Yup, boolean short circuit :) I'll make it a little more clear